Packs/Malwr/Integrations/Malwr/Malwr.py

```diff
@@ -0,0 +1,317 @@
+import hashlib
+import re
+
+import demistomock as demisto  # noqa: F401
+import requests
+from bs4 import BeautifulSoup
+from CommonServerPython import *  # noqa: F401
+
+MAIN_URL = 'https://malwr.com'
+STATUS_URL = '/submission/status/%s/'
+RESULT_URL = '/analysis/%s/'
+MD5_PREFIX_STR = 'with MD5 '
+SUPPORTED_COMMANDS = ['Submit', 'Status', 'Result', 'Detonate']
+DETONATE_DEFAULT_TIMEOUT = 600
+DETONATE_POLLING_INTERVAL = 10
+
+
+def md5(fname):
+    hash_md5 = hashlib.md5()  # nosec B324
+    with open(fname, 'rb') as f:
+        for chunk in iter(lambda: f.read(4096), b''):
+            hash_md5.update(chunk)
+
+    return hash_md5.hexdigest()
+
+
+def get_file_path(file_id):
+    filepath_result = demisto.getFilePath(file_id)
+    if 'path' not in filepath_result:
+        demisto.results(f'Error: entry {file_id} is not a file.')
+        return
+
+    return filepath_result['path']
+
+# The Malwar API from https://github.com/PaulSec/API-malwr.com
+
+
+class MalwrAPI:
+    """
+        MalwrAPI Main Handler
+    """
+    headers = {
+        'User-Agent': "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:41.0) "
```

Comment:
Can this be a constant?
It's pretty old (Firefox 41.0 is several years old...)

Reply:
I'll make it a class attribute.